common/archiver/filestore/historyArchiver.go

```diff
@@ -236,7 +237,8 @@ func (h *historyArchiver) Get(
 		return nil, &shared.EntityNotExistsError{Message: archiver.ErrHistoryNotExist.Error()}
 	}
 
-	encodedHistoryBatches, err := readFile(filepath)
```

Comment:
Can we keep this readFile file method and just add a #nosec tag on the ioutil.ReadFile in the readFile method? Even though all readFile does is ioutil.ReadFile I prefer having all file operations live in one place.

Reply:
+1 for this, it's possible we want to change this function later and add some custom logic.

Reply:
@carlcortright can you change it back to readFile helper method?  We should just have a strong worded comment on the helper method considering all usages are already audited and we can audit future usages during code reviews.

Reply:
Yup will do

Reply:
Just updated. Sorry for the delay.